pkg/cmd/templates.go

```diff
@@ -38,24 +37,15 @@ func WrappedLocalFlagUsages(cmd *cobra.Command) string {
 // commands to set values for request parameters. The string is wrapped to the
 // terminal's width.
 func WrappedRequestParamsFlagUsages(cmd *cobra.Command) string {
-	var sb strings.Builder
-
-	// We're cheating a little bit in thie method: we're not actually wrapping
-	// anything, just printing out the flag names and assuming that no name
-	// will be long enough to go over the terminal's width.
-	// We do this instead of using pflag's `FlagUsagesWrapped` function because
-	// we don't want to print the types (all request parameters flags are
-	// defined as strings in the CLI, but it would be confusing to print that
```

Comment:
just to confirm that we are ok to show type `string` to every request param flag now??

Reply:
Was thinking about this more and I may remove it. It's still not that helpful in the grand scheme

Reply:
I think we may want to rethink some of this user facing output in a follow-on PR. This PR in general is a _big_ win, but it may change the optimal shape of `--help`

Reply:
I'll undo this. It'll be complicated to improve it as part of this PR, the underlying code in the pflags library is a bit tricky to work through